birdhouse/scripts/backup-datavolume.sh

```diff
@@ -1,15 +1,15 @@
 #!/bin/sh -x
-# Backup DATA_VOL_NAME to BACKUP_OUT_DIR/DATA_VOL_NAME.tgz
+# Backup DATA_VOL_DIR to BACKUP_OUT_DIR/DATA_VOL_DIR.tgz
 
-DATA_VOL_NAME="$1"; shift
+DATA_VOL_DIR="$1"; shift
 BACKUP_OUT_DIR="$1"; shift
 
 docker run --rm \
-  --name backup_data_vol_$DATA_VOL_NAME \
```

Comment:
Don't change this script.  This script is to backup a data volume, now we use bind mount.  We will simply not use this script and use something else for bind mount.

Keep this script as-is for other data volume backup in the future.

Reply:
Done